llvm/test/CodeGen/AMDGPU/lds-no-realign-allocated-variables.ll

```diff
@@ -0,0 +1,18 @@
+; RUN: opt -S -mtriple=amdgcn-- -passes=amdgpu-lower-module-lds < %s | FileCheck %s
+
+; Can't have a second variable without absolute_symbol showing it is realigned as
+; there is a fatal error on mixing absolute and non-absolute symbols
+
+; CHECK: @lds.dont_realign_undef = internal addrspace(3) global i64 undef, align 2, !absolute_symbol !0
+; CHECK: @lds.dont_realign_poison = internal addrspace(3) global i64 poison, align 2, !absolute_symbol !0
+@lds.dont_realign_undef = internal addrspace(3) global i64 undef, align 2, !absolute_symbol !0
```

Comment:
no reason to have the undef one, or separate names. just have one normal variable with undef

Reply:
yeah, race condition between review requests and updating the branch